torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -192,6 +192,7 @@ class CppMicroGemmConfig:
     vec_isa_cls: type[VecISA]
     register_blocking: GemmBlocking
     extra_check: Optional[Callable[..., bool]] = None
+    trans_b: bool = False
```

Comment:
Do we need to add this flag into `CppMicroGemmConfig`? By current usage, this flag used to create `CppMicroGemmFP32Vec` directly.

Reply:
Remove `trans_b` from `CppMicroGemmConfig`.